protos/dcs/coalition/coalition.proto

```diff
@@ -23,6 +23,100 @@ service CoalitionService {
    */
   rpc GetMainReferencePoint(GetMainReferencePointRequest)
       returns (GetMainReferencePointResponse) {}
+
+  // https://wiki.hoggitworld.com/view/DCS_func_getPlayers
+  rpc GetPlayers(GetPlayersRequest) returns (GetPlayersResponse) {}
+}
+
+message AddGroupRequest {
+	dcs.common.Country country = 1;
+	dcs.common.GroupCategory category = 2;
+	oneof template {
+	  GroundGroupTemplate ground_template = 3;
+	  ShipGroupTemplate ship_template = 4;
+	  HelicopterGroupTemplate helicopter_template = 5;
+	  PlaneGroupTemplate plane_template = 6;
+	}
+
+	message GroundGroupTemplate {
+		optional uint32 group_id = 1;
+		optional bool hidden = 2;
+		optional bool late_activation = 3;
```

Comment:
We might be able to simplify booleans by making `false` the default as we don't need them to be `optional` then. I think those two are already `false` by default, so I think we can ditch the `optional`?

Reply:
Good point.